build/ks10/include.tcl

```diff
@@ -92,6 +92,7 @@ proc its_switches {} {
 proc make_ntsddt {} {
     respond "*" ":midas dsk0:.;@ ddt_system;ddt\r"
     respond "cpusw" "3\r"
+    respond "dsktp=" "1\r"
```

Comment:
Since dsksw=0, I believe the value of dsktp doesn't matter.  Maybe the dsktp question shouldn't come up at all?

Reply:
I'll enter a separate ticket on this.